lib/object_3d.dart

```diff
@@ -183,11 +216,13 @@ class _ObjectPainter extends CustomPainter {
     for (int i = 0; i < face.length; i++) {
       double x, y;
       if (i < face.length - 1) {
-        x = vertices[face[i + 1] - 1].x.toDouble();
-        y = vertices[face[i + 1] - 1].y.toDouble();
+        final Vector3 iV = vertices[face[i + 1] - 1];
+        x = iV.x.toDouble();
+        y = iV.y.toDouble();
       } else {
-        x = vertices[face[0] - 1].x.toDouble();
-        y = vertices[face[0] - 1].y.toDouble();
+        final Vector3 iV = vertices[face[0] - 1];
```

Comment:
This one neither. `final iV = vertices[face[0] - 1];`  suffices

Reply:
https://github.com/arnovanliere/object_3d/pull/2#discussion_r1523723082

Reply:
done.